### DISCUSSION TITLE
Error in Uploadthing package in production env.          SyntaxError: Unexpected token 'A', "An error o"... is not valid JSON

### DISCUSSION THREAD

**Discussion by senaa07** [vercel/next.js#58867] [Help]
### Summary

Something went wrong. Please contact UploadThing and provide the following cause: SyntaxError: Unexpected token 'A', "An error o"... is not valid JSON

core.ts
<img width="718" alt="image" src="https://github.com/vercel/next.js/assets/101617549/67cf1acf-82ab-4669-82ab-7cc2fdfb05f3">

route.ts
<img width="718" alt="image" src="https://github.com/vercel/next.js/assets/101617549/8443e6d6-1572-4218-ac82-ef291d09b5d5">

The app is working perfectly in local environment. In production while i upload file i am getting the same error again and again. But the file is getting uploaded in uploadthing dashboard. Can anyone help me to solve this.

**Comment by MalaRuze** [comment]
Hi @X-sena-X  , I get the same error. I investigated further Vercel logs and discovered that the Vercel serverless function is timing out after 10s.  So uploadthing function cannot finish fast enough to fit in Vercel free plan limitations. Which sucks, I would love to use uploadthing with Vercel. There must be some other solution. Did you manage to solve it?

**Comment by MalaRuze** [comment]
Try updating uploadthing packages to the latest release version https://github.com/pingdotgg/uploadthing/releases 

- @uploadthing/react@6.0.2
-  uploadthing@6.0.3

It fixed the issue for me:)

**Comment by senaa07** [comment]
Hi, @MalaRuze thanks for letting it know. I solved this issue by doing the same.
